presentation/backend/src/presentation-backend/PresentationIpcHandler.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module RPC
+ */
+
+import { IpcHandler } from "@bentley/imodeljs-backend";
+import { presentationIpcChannel, PresentationIpcInterface, RulesetVariableJSON, SetRulesetVariableParams } from "@bentley/presentation-common";
+import { Presentation } from "./Presentation";
+
+/** @internal */
+export class PresentationIpcHandler extends IpcHandler implements PresentationIpcInterface {
+  public channelName = presentationIpcChannel;
+
+  public async setRulesetVariable(params: SetRulesetVariableParams<RulesetVariableJSON>): Promise<void> {
+    const { variable } = params;

```

Comment:
Maybe destructure `clientId` and `rulesetId` as well?

Reply:
Changed